documentation/automation.md

```diff
@@ -0,0 +1,50 @@
+# Volunteers for Salesforce Automation Inventory
+
+## Key Workflows
+
+This section describes the key workflows for developers, QEs, and technical
+writers.
+
+| Workflow                   | Flow                 | Org Type       | Managed | Namespace       | NPSP |
+| -------------------------- | -------------------- | -------------- | ------- | --------------- | ---- |
+| Development                | `dev_org`            | dev            |         |                 |      |
+| Development (Namespaced)   | `dev_org_namespaced` | dev_namespaced |         | `GW_Volunteers` |      |
+| QA                         | `qa_org`             | dev            |         |                 |      |
+| Regression                 | `regression_org`     | regression     | ✔       |                 |      |
```

Comment:
@davidmreed - Has Regression changed? Last time I tested in it, regression_org flow also had the GW_Volunteers__ namespace.

Reply:
@kselvocki Right - we're distinguishing between _orgs_ that are namespaced (everything in the org gets the namespace), and managed installations (which have the namespace on the packaged components only).